tests/outputs/abi.tc.Visible.imagery_clean/20200918_195020_AL202020_abi_goes-16_Visible_110kts_100p00_res1p0-arB03Ref-clean.png.yaml

```diff
@@ -28,6 +28,8 @@ pressure: 952.47
 product_filename: $GEOIPS_OUTDIRS/preprocessed/tcwww/tc2020/AL/AL202020/png_clean/Visible/goes-16/20200918_195020_AL202020_abi_goes-16_Visible_110kts_100p00_res1p0-arB03Ref-clean.png
 recenter_type: B03Ref
 sector_type: tc
+source_file_names:
+- OR_ABI-L1b-RadF-M6C03_G16_s20202621950205_e20202621959513_c20202621959570.nc
```

Comment:
Do you have a copy/paste of the output from running this test?

Reply:
See revised PR overview. Full script output is shown!

Reply:
Thanks!